src/components/_modal/pages/receipt/Receipt.page.js

```diff
@@ -51,26 +51,26 @@ export function Receipt({
   const isOrdering = orderIsLoading || articleOrderIsLoading || delay;
   const isOrdered =
     !!orderData?.submitOrder?.orderId ||
-    articleOrderData?.elba?.placeCopyRequest?.ok;
+    articleOrderData?.elba?.placeCopyRequest?.status === "OK";
```

Comment:
Retur-børgen for `elba` (har kun status)  og `submitOrder` (har status, ok, message) burde måske have samme form, for enrettetheds skyld (dog nok ikke relevant ift. den her story)

Reply:
enig og enig.